client.js

```diff
@@ -51,8 +51,9 @@ bot.on('ready', () => {
 
 bot.on('message', function (message) {
     // console.log(message);
-    // ignore bots or messages from any text channel that isn't the Command Channel
-    if (message.author.bot || (message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL)){
```

Comment:
Now that we don't short-circuit on messages in other channels, won't it hit the `else` below on messages without "!" and then spam channels when people talk?

Reply:
You're right, fixed.